src/js/Templates/Shared/Image.js

```diff
@@ -83,6 +83,7 @@ class Image extends React.Component {
         // Refresh image after onPutFile in case of error
         if (this.state.error && this.props.refresh) {
             this.setState({error: false});
+            this.props.refresh = false
```

Comment:
I think its bad practice to modify a components props in the render function. Is this change necessary? This statement should not execute since state.error is set to false. The app.refresh prop should be set to false in reducers.js on the next state change.